portfolio/src/main/webapp/comments.html

```diff
@@ -22,25 +22,33 @@
     <div  id="content" class="fade-in">
 
       <form class="center" action="/new-comment" method="POST">
-        Leave a comment!
+        <label for="comment">Leave a comment!</label>
```

Comment:
Good job adding labels :) make sure you add id attributes to your textarea elements. This is what will link the two together. So `<textarea id="comment" name="comment" ...>`